analyzers/src/SonarAnalyzer.CSharp/Rules/UseAwaitableMethod.cs

```diff
@@ -158,7 +159,8 @@ private static WellKnownExtensionMethodContainer BuildWellKnownExtensionMethodCo
             }
             if (original == awaitableRoot && result is ExpressionSyntax resultExpression)
             {
-                result = SyntaxFactory.AwaitExpression(resultExpression);
+                result = SyntaxFactory.ParenthesizedExpression(
```

Comment:
Is is possible to add a test that fails if this line is not changed to add parentheses around the replacement?

Reply:
Maybe add a quick comment there to capture this?
On why there are parentheses.

Reply:
I'll merge this to make it to the next release. Let's add a comment in one of the next PRs. 
I'll keep a task on my calendar to not forget.